index.html

```diff
@@ -5,23 +5,27 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="core.css" media="screen and (min-width: 768px)">
     <script src="https://kit.fontawesome.com/936703f17d.js" crossorigin="anonymous"></script>
+   
     <title>Portfolio</title>
 </head>
 <body>
     <nav class="nav">
         <a>
             <img src="logo.png" alt="logo"
-                width="30px" height="20px">
+                width="30px" height="20px" class="logo">
         </a>
-        <i class="fa-solid fa-bars"></i>
+        <div class="hamburger">
+            <i class="fa-solid fa-bars"></i>
+            <i class="fa-solid fa-xmark"></i>
+        </div>
         <ul class="nav-list">
-            <li class="animation"><a href="#">Portifolio</a></li>
-            <li class="animation"><a href="#">About</a></li>
-            <li class="animation"><a href="#">Contact</a></li>
-            <li class="animation"><a href="#"><i class="fa-regular fa-envelope"></i></a></li>
+            <li class="animation nav-item"><a href="#">Portifolio</a></li>
+            <li class="animation nav-item"><a href="#">About</a></li>
+            <li class="animation nav-item"><a href="#">Contact</a></li>
+            <li class="animation nav-item envelope"><a href="#"><i class="fa-regular fa-envelope"></i></a></li>
```

Comment:
- [ ] according to the requirements when a user clicks on the link the page should jump to the part of the page the link is referring to
- [ ] in addition to that the navigation menu should close again